src/components/pages/PendingApplications/MentorModal.js

```diff
@@ -0,0 +1,108 @@
+import { List, Tag } from 'antd';
+
+const MentorModal = ({ applicant }) => {
+  return (
+    <>
+      <div className="profile-intro">
+        <div className="image-container">
+          <img
+            src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
```

Comment:
Be careful about keeping links from third party sites as sources for things. You never know if they will switch out the image with the same name or something else could happen. I imagine this is not going to stay in the MVP. I would find an alternative image source (to reference locally? There is a public folder)